tests/test_indexing.py

```diff
@@ -13,13 +13,12 @@ def test_full_window():
     with rasterio.open('tests/data/RGB.byte.tif') as src:
         assert src.window(*src.bounds) == tuple(zip((0, 0), src.shape))
 
-def test_window_exception():
+def test_window_no_exception():
     with rasterio.open('tests/data/RGB.byte.tif') as src:
         left, bottom, right, top = src.bounds
         left -= 1000.0
         try:
             _ = src.window(left, bottom, right, top)
-            assert False
-        except ValueError:
             assert True
-
+        except ValueError:
```

Comment:
@sgillies try / catch doesn't seem necessary here.  You are just verifying that src.window() executes.    (and if it throws something other than ```ValueError``` we'd want this test to fail here)